lib/velum/salt.rb

```diff
@@ -70,6 +70,23 @@ def self.accept_minion(minion_id: "")
       JSON.parse(res.body)["return"].first["data"]["return"]["minions"]
     end
 
+    # Removes a minion from the cluster
+    def self.remove_minion(minion_id: "")
+      res = perform_request(endpoint: "/", method: "post",
+                            data: { client: "wheel",
+                                    fun:    "key.delete",
+                                    match:  minion_id })
+    end
+
+    # Rejects a minion from the cluster
+    def self.reject_minion(minion_id: "")
+      res = perform_request(endpoint: "/", method: "post",
+                            data: { client: "wheel",
+                                    fun:    "key.reject",
+                                    arg:    "--incldue-accepted",
```

Comment:
I don't think you can pass args like this (besides, there's a typo in `include`). I think this should be something like:

```
      res = perform_request(endpoint: "/", method: "post",
                            data: { client: "wheel",
                                    fun:    "key.reject",
                                    include_accepted: True,
                                    match:  minion_id })
```

Reply:
Thanks

Reply:
Will remove this line since we are only rejecting no accepted nodes.